.vscode/launch.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
I recommend not to push the entire .vscode folder into git.
It contains local configurations which can be different from user to user. 

It's better to write it in the documentation instead and everyone who will want will add them manually.

Reply:
but it does not contains personal settings i debug with vscode, i must have it

Reply:
it does, your debugging profile is personal, I use something else for example and by pushing .vscode folder you overriding my settings.
I suggest put it in .gitignore and then everyone will have their own settings, 

This folder also includes project settings, for example I've set that I dont see any of the `d.ts` files in the explorer, if I'd push it you won't see them either.

Reply:
ok you got it , i want to sleep ;-)